fern/docs/pages/docs/quickstart.mdx

```diff
@@ -64,7 +64,7 @@ Examples:
 
 ```fern init --openapi https://petstore3.swagger.io/api/v3/openapi.json```
 
-```fern init --openapi ../apis/openapi.yml```
+```fern init --openapi ../fern/openapi/openapi.yml```
```

Comment:
This filepath doesn't make sense to me. The filepath of _FERN_REPO_NAME_/fern/openapi/_OPENAPI_FILENAME is created _by_ running `fern init --openapi OPENAPI_FILENAME`. Before this, it would exist at a different path, then copied to that folder.